examples/hex-game/tests/hex_game.rs

```diff
@@ -34,8 +34,15 @@ async fn hex_game() {
         .await;
 
     let block = certificate.inner().block();
-    let message_id = block.message_id_for_operation(0, 0).unwrap();
-    let description = ChainDescription::Child(message_id);
+    let description = block
+        .created_blobs()
+        .into_iter()
+        .filter_map(|(blob_id, blob)| {
+            (blob_id.blob_type == BlobType::ChainDescription)
+                .then(|| bcs::from_bytes::<ChainDescription>(blob.content().bytes()).unwrap())
```

Comment:
Check the operation results instead, and make the operation return the child chain ID as a result? (Also below.)